cadquery/cq.py

```diff
@@ -3189,28 +3176,39 @@ def sweep(
             auxSpine,
         )  # returns a Solid (or a compound if there were multiple)
 
-        newS: T
-        newS = self._combineWithBase(r, combine)
-        if clean:
-            newS = newS.clean()
-        return newS
+        return self._combineWithBase(r, combine, clean)
 
     def _combineWithBase(
-        self: T, obj: Shape, combine_mode: Union[bool, str] = True
+        self: T,
+        obj: Union[Shape, Iterable[Shape]],
+        mode: CombineMode = True,
+        clean: bool = False,
     ) -> T:
         """
         Combines the provided object with the base solid, if one can be found.
         :param obj: The object to be combined with the context solid
-        :param combine_mode: The mode to combine with the base solid (True, False or "cut")
+        :param mode: The mode to combine with the base solid (True, False or "cut")
```

Comment:
We should maybe document the fact that we can pass `cut="a"` and `cut="s"` here and in all the method that use combine?